### PR TITLE
[WIP] Allow purchasing multiple same CP items

### PR DESCRIPTION
**Pull request by davismj**
If you try to purchase multiple of the same CP item it will not let you whether or not the item is rare. It should allow this behavior.

**Test:**
- [ ] Purchasing multiple non-rare items
- [ ] Purchasing multiple rare items
- [ ] Trading EXP rings

### CODE REVIEW

**Comment by Wiggo32** [comment]
I forgot about this issue until you just brought it up. But, yes you should be able to purchase multiple Non-Rare Items. Found this problem out while trying to synth a HQ body piece a long time ago. They only gave me 1 try! P.S. I've done it on retail since then just to make sure I wasn't crazy.

File: scripts/globals/conquest.lua [hunk 1]
```diff
@@ -3,1388 +3,1360 @@
...
-
-nationAlly = 3;
+require("scripts/globals/keyitems")
+require("scripts/globals/missions")
+require("scripts/globals/common")
+
+dsp = dsp or {}
+dsp.conquest = {}
+
+dsp.conquest.NATION_SANDORIA = 0
```
Comment (nfaletra) [inline scripts/globals/conquest.lua]: Should this be divided into dsp.nation, dsp.region, and dsp.conquest? You assign the same values for dsp.conquest.CONQUEST_TALLY down below and that is super gross.

Also, assuming dsp.nation/dsp.region/dsp.conquest is defined in this file, just do dsp.nation= { NATION_SANDORIA = 0, NATION_BASTOK = 1, etc. }
More C-style enum and consistent with other tabled values.

You can still continue to define your functions down below outside the initializations, too.

**Comment by helixhamin** [comment]
@davismj So, should this one be closed, or readjusted?

**Comment by davismj** [comment]
I need to rebase / start over.

**Comment by takhlaq** [comment]
i'll close for now otherwise i'll keep rechecking this

**Pull request closed without merging**